src/assets/styles/layout.scss

```diff
@@ -53,7 +53,7 @@
 }
 
 .section-container {
-  padding: 1.25rem 1.25rem;
+  padding: 1.25rem 1.875rem;
```

Comment:
This is now inconsistent with Create UI. Did UX/UI ask for this?

https://github.com/bcgov/business-create-ui/blob/main/src/assets/styles/base.scss#L304

Reply:
Indeed :( 
Designs call for a consistent 30px padding on the left and right. 
What is the spacing in Create designs?

Reply:
OK.

I don't see exact values in recent designs for Create UI so I implemented something that matched the look of their designs. 🤷‍♂️ 

Resolving this conversation.